web/packages/teleport/src/SessionRecordings/view/RecordingPlayer.tsx

```diff
@@ -88,21 +89,26 @@ export function RecordingPlayer({
     );
   }
 
+  const sshPlayer = (
+    <SshPlayer
+      ref={ref}
+      onTimeChange={onTimeChange}
+      onToggleSidebar={onToggleSidebar}
+      sid={sessionId}
+      clusterId={clusterId}
+      durationMs={durationMs}
+      onToggleTimeline={onToggleTimeline}
+    />
+  );
+
+  if (!RECORDING_TYPES_WITH_METADATA.includes(recordingType)) {
+    // For recording types without metadata (e.g., database), render the legacy SshPlayer directly.
+    return sshPlayer;
```

Comment:
The version from this branch ends up not rendering the player at full height. Perhaps it needs to be wrapped in `<Container>`?

(left is current master, right is this branch)

<img width="2559" height="1415" alt="db-session" src="https://github.com/user-attachments/assets/dc77f865-e3af-44c1-9bd3-eea6cb5d327b" />